VimeoNetworking/VimeoNetworking/ResponseCache.swift

```diff
@@ -64,24 +73,24 @@ final internal class ResponseCache
     {
         let key = request.cacheKey
         
-        self.memoryCache.removeResponseDictionaryForKey(key)
```

Comment:
Gotcha, nit: wondering if this should not just be setValueForKey, valueForKey, removeValueForKey etc?